meta-printnanny/recipes-core/psplash/psplash_git.bbappend

```diff
@@ -7,6 +7,7 @@ SRC_URI += "\
     file://10-fb0-systemd.rules \
 "
 SPLASH_IMAGES:rpi = "file://psplash-printnanny-img.h;outsuffix=printnanny"
```

Comment:
I think we can just use the distro override here since we're not changing the splash per machine. The base recipe had a Raspberry Pi logo on RPI, but we use the same one everywhere. 

```
SPLASH_IMAGES:printnanny = "file://psplash-printnanny-img.h;outsuffix=printnanny"
```